### PR TITLE
refact(cspc-operator): added cspc crd yaml to cspc-operator

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

This PR adds CSPC CRD yaml to cspc-operator for installation. The PR also includes a script to delete the CRD if present and apply the cspc-operator yaml.
**Note: Before applying the above operator yaml the old CRD needs to be deleted**

### CODE REVIEW

**Comment by vishnuitta** [comment]
lets have a script to delete old CSPC crd yaml if no CR exists and applies the new one

File: k8s/cspc_install.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+x=$(kubectl get crd cstorpoolclusters.openebs.io -o jsonpath="{.spec.scope}" 2>&1 )
+
+if [[ $x == 'Cluster' ]]; then
+    kubectl delete crd cstorpoolclusters.openebs.io
+fi
+
+echo "Installing cspc operator"
+
+kubectl apply -f cspc-operator.yaml
```
Comment (vishnuitta) [inline k8s/cspc_install.sh]: better to change it to full raw github link

Comment (vishnuitta) [reply k8s/cspc_install.sh]: again, it depends on our doc steps..
if our doc says to download and run, then, this should be fine

Comment (shubham14bajpai) [reply k8s/cspc_install.sh]: added the raw github link instead as there is nothing that user can customize in this yaml

**Pull request merged**